app/models/postcode_local_restriction_search.rb

```diff
@@ -42,6 +42,10 @@ def invalid_postcode?
     %w[invalidPostcodeFormat fullPostcodeNoMapitValidation].include?(error_code)
   end
 
+  def no_restriction_information_england_postcode?
```

Comment:
I wonder if this could be called `restriction?`.  Then when we call it, it would read as `@search.restriction?` or `!@search.restriction?`

I don't think we need to specify that it's only for England in the name as the whole checker is only for England.

Reply:
@leenagupte I changed this to `no_restriction?` as I struggled with `!@search.restriction?` due to the devolved nation needing to be true for `location_lookup.data.first&.england?`... also naming sort of works due to being similar to `no_information?`. I think Kevin will somewhat overwrite this anyway with his refactor.